services/codecov/badge-100.svg

```diff
@@ -0,0 +1,23 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="122" height="20">
```

Comment:
I was a little confused by these files at first, though now realize they're being used as mock/test data. In similar scenarios elsewhere in the codebase where we need the svg xml for a test, we've done so via an inline string in a js file (for example [here](https://github.com/badges/shields/blob/612a43aaa9cd8435c4165a7d2ebd5bc44838a0da/services/snyk/snyk-test-helpers.js#L3-L6) or [here](https://github.com/badges/shields/blob/612a43aaa9cd8435c4165a7d2ebd5bc44838a0da/lib/svg-helpers.spec.js#L8)

I'd prefer to see something similar done here, or at least move these files under a subdirectory that clearly indicates their purpose (i.e. `mocks/badge-100.svg` ) although I don't feel _too_ strongly about it